examples/systemcalls/systemcalls.c

```diff
@@ -148,6 +132,54 @@ bool do_exec_redirect(const char *outputfile, int count, ...)
  *   
 */
 
+    int kidpid, status;
+    int wait_ret=0;
+    int fd = open(outputfile, O_WRONLY|O_TRUNC|O_CREAT, 0644);  //open file to be redirected to 
```

Comment:
@saloni1307 think about how you could combine logic with the function above to avoid duplication.